config/class-sitedetailsindex.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Automattic\VIP\Config;
+
+class SiteDetailsIndex {
```

Comment:
Minor: underscores

```suggestion
class Site_Details_Index {
```

All references below and elsewhere would need to be updated as well. And we should rename the file to `class-site-details-index.php`.